### PR TITLE
Move code schema to version 3.0.0

### PR DESCRIPTION
**Pull request by gtuk**
# Pull Request

### Description
Replaced all version 2.0.0 code schema references with version 3.0.0

### CODE REVIEW

File: src/main/resources/code-xsd/GI4X-basis-2.0.0.xsd [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
-<!-- THIS FILE WAS PROVIDED BY GKV DATENAUSTAUSCH AT https://www.gkv-datenaustausch.de/leistungserbringer/digitale_gesundheitsanwendungen/digitale_gesundheitsanwendungen.jsp -->
-<!-- AND IS COPYRIGHTED BY ITS ORIGINAL AUTHORS -->
-<!-- IT HAS NOT BEEN MODIFIED OTHER THAN INCLUDING THIS NOTICE -->
-
-<!-- Mit XMLSpy v2020 rel. 2 (x64) (http://www.altova.com), Stand vom 01.07.2020 - Entwurfstatus -->
+<!-- Mit XMLSpy v2013 sp1 (http://www.altova.com) von Claudia Pawel (GKV-Spitzenverband) bearbeitet -->
+<!-- edited with XMLSpy v2007 sp2 (http://www.altova.com) by Andreas Meier (ITSG GmbH) -->
 <xs:schema xmlns:dd="GI4X:/data-dictionary" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:GI4X-basis="GI4X:/xml-schema/GI4X-basis/2.0" targetNamespace="GI4X:/xml-schema/GI4X-basis/2.0" elementFormDefault="qualified" attributeFormDefault="unqualified" version="2.0.0">
 	<xs:annotation>
 		<xs:documentation>
```
Comment (fongie) [inline src/main/resources/code-xsd/GI4X-basis-2.0.0.xsd:6]: this file stays on 2.0.0 is that intended?

Comment (gtuk) [reply src/main/resources/code-xsd/GI4X-basis-2.0.0.xsd:6]: Yes this file didn't change, except for the comment

**Comment by fongie** [comment]
Im trying to give it a bit of a testrun and failing miserably hehe, have you run this and it worked? In that case I don't really need to check it. Im not getting the health insurance information to populate in the client, after changing the Kranken...xml file to 3.0 everywhere, all the health insurance information come out as null after parsing

**Comment by fongie** [comment]
nevermind, I got it working :) nice!

**Review by fongie — APPROVED** [review]

**Pull request merged**